vm/src/vm/runners/cairo_pie.rs

```diff
@@ -52,15 +54,17 @@ pub enum BuiltinAdditionalData {
     None,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+#[derive(Serialize, Clone, Debug, PartialEq, Eq)]
 pub struct CairoPie {
     pub metadata: CairoPieMetadata,
+    #[serde(serialize_with = "serde_impl::serialize_memory")]
     pub memory: CairoPieMemory,
     pub execution_resources: ExecutionResources,
     pub additional_data: HashMap<String, BuiltinAdditionalData>,
+    pub version: CairoPieVersion,
```

Comment:
I don't know if it is possible, but can we use a String type for the version field.
Plus add a constant with the CairoPieVersion String, so it is easy to change

Reply:
While it is possible, it'd need to be initialized every time the CairoPie object is created. Having it as it is now this cannot happen because the version field is zero-sized.

The constant is easy. Fixed.

Reply:
nice